doc/rados/operations/cache-tiering.rst

```diff
@@ -432,11 +432,11 @@ disable and remove it.
 #. Change the cache mode to ``proxy`` so that new and modified objects will 
    flush to the backing storage pool. ::
 
-	ceph osd tier cache-mode {cachepool} proxy
+	ceph osd tier cache-mode {cachepool} readproxy
```

Comment:
It has been resurrected in https://github.com/ceph/ceph/pull/45409.

Reply:
Alright, closing this.